pkg/storage/minio/getter.go

```diff
@@ -62,6 +65,7 @@ func (v *storageImpl) Zip(ctx context.Context, module, vsn string) (storage.Size
 	}
 	oi, err := zipReader.Stat()
 	if err != nil {
+		zipReader.Close()
```

Comment:
Why are we not using a defer call here?

Reply:
Can we also add zipReader.close when GetObject returns an error as well? This is the only one change that I require of this PR and we are good to go.

Thanks for making these changes.

Reply:
> Why are we not using a defer call here?

We can't defer a `Close` for `zipReader`, since it is returned and read elsewhere. All instances I was able to fine appear to properly close the returned value, so the only time we need to close it is on error.

> Can we also add zipReader.close when GetObject returns an error as well?

I specifically avoided that to avoid calling `Close` on `nil`.

Reply:
Thanks a lot. I realized after reading through the comment left in description. I did not realize GetObject can return nil as well on error (I wish that was not the case though)